GeosCore/drydep_mod.F90

```diff
@@ -3347,11 +3423,29 @@ FUNCTION AERO_SFCRSII( K, II, PRESS, TEMP, USTAR, RHB, W10, Input_Opt ) &
     NR = INT((( Input_Opt%SALC_REDGE_um(2) - Input_Opt%SALA_REDGE_um(1) ) &
          / DR ) + 0.5e+0_f8 )
 
+    ! drydepRadius
+    SpcId = NTRAIND(K)
+    SpcInfo => State_Chm%SpcData(SpcId)%Info
+
+    drydepRadius = A_RADI(K)
+
+    ! Coarse seasalt
+    IF ((SpcInfo%Name .EQ. 'NITS') .OR. (SpcInfo%Name .EQ. 'SALC') & 
```

Comment:
Would also recommend using species ID flags instead of string comparisons here.

Reply:
Thanks @yantosca ! I agree with all your suggestions! Should I submit another pull request after updating all these changes?

Besides, would it be possible to have an example of GEOS-Chem code snippet using species ID flags so that I could follow easier?

Reply:
@YanshunLi-washu: You can push changes to this PR.  Sorry for the delay in responding.